modules/schema/src/main/resources/lucuma/odb/graphql/OdbSchema.graphql

```diff
@@ -10828,17 +10828,13 @@ type TargetEnvironment {
   ): [GuideEnvironment!]!
 
   """
-  The guide star(s) and related information
+  The guide target(s) and related information.
+  If a guide target has been set via `guideTargetName`, that target will be
+  returned. If it not found or not usable, an error will be returned.
+  If no guide target has been set, or it has been invalidated by observation/target
+  changes, Gaia will be searched for the best guide target available.
   """
-  guideEnvironment(
-    """
-    If this is set to false, a null will be returned if a guide star is not
-    already set or the current guide star has been invalidated by changes.
-    If this is set to true, gaia will be searched to find the best guide star
-    in the above scenario.
-    """
-    lookupIfUndefined: Boolean! = true
-  ): GuideEnvironment
+  guideEnvironment: GuideEnvironment!
```

Comment:
Why was the parm removed?

Reply:
Because this is used by Navigate, and it always wants the odb to find a default value if one isn't set. The parameter would only have been set to false by Explore.